Add files via upload

Martxv opened the pull request:
Dopo aver caricato la pagina web, ne ho constatato il malfunzionamento. Questa è la versione corretta e, spero, funzionante.

strumet commented:
Ciao Alessandro,
per poter aggiungere i tuoi file al repository del corso è necessario che siano formalmente corretti.
Per questo dovresti rimuovere la cartella ex/vectors/SVG spostando i file in essa contenuti nella cartella ex/vectors.
Vanno inoltre rispettate le indicazioni sulla denominazione dei file raster (IMG_6851.jpg) e rimossi i file non richiesti (MO-D_2_CAMERA-SEZIONE.pdf).
Dopo aver sistemato i file è sufficiente effettuare un commit ed un push (questo pull request resta aperto e verrà automaticamente aggiornato)

strumet commented:
file aggiunti :+1:

Martxv committed:
Add files via upload

Dopo aver caricato la pagina web, ne ho constatato il malfunzionamento. Questa è la versione corretta e, spero, funzionante.
